libc/dirent/main.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * Main entry point.
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Arkadiusz Kozlowski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include "common.h"
+#include "unity_fixture.h"
+
+/* no need for forward declarations, RUN_TEST_GROUP does it by itself */
```

Comment:
I think convention to start comments that contain more than 1 word with upper letter.
```suggestion
/* No need for forward declarations, RUN_TEST_GROUP does it by itself */
```